### ISSUE TITLE
Project status and npm publish?

### ISSUE THREAD

**Issue by meltuhamy** [Profiscience/ko-hot-loader#1]
Hello, just wondering what's the status of this project / if you still use it? 

Also, have you published this on npm?

**Comment by caseyWebb** [comment]
I don't believe it's functional at the moment to be honest. I've implemented HMR in our business app, and this was an attempt to extract it, but it's poorly tested and I haven't looked at it in quite a bit. I'd wouldn't mind revisiting it, however.
